NamingPolicyCheck.py

```diff
@@ -89,7 +89,8 @@ def check_binary(self, pkg):
 # if somone as a elegant solution, I will be happy to implement and test it.
 
 check.add_check('xmms', '^xmms(-|$)', '^/usr/lib(64)?/xmms/')
-check.add_check('python', '^python(-|$)', '^/usr/lib(64)?/python[1-9](-[1-9])?')
+check.add_check('python2', '^python(2)?(-|$)', '^/usr/lib(64)?/python[1-2](-[1-9])?')
```

Comment:
I don't know if it's reasonable to expect `python1.y` named as `python2` subpackage. 😜 

Could this check be made better to map major versions, such that when `python4` becomes a thing, it will be picked up correctly by policy?

Reply:
not easily as it expect as first argument unique name for each check, and I kept the 1 there just for the fun of it as it can read the regular python- which were used before as python2 (and arguably even python1 arguments).

Reply:
Perhaps we should leave this off the table for now, since it's a bit ugly and not easy to future-proof right now...